LastWork(newWork)/Views/new-evaluation/newEvaluationPage.js

```diff
@@ -316,35 +230,163 @@ const DropdownSelector = function (option) {
             label: 'Unit Testing',
             name: 'unitT',
         }],
-    }];
+    }],
+};
+
+interviewApp.CandidateDetailsForm = {
+    init() {
+        return `
+    <input id="CandName" class="textbox" type="text" placeholder="Candidate" name="CandName">
+    <input id="IntervName" class="textbox" type="text" placeholder="Interviewer" name="IntervName">
+    <input id="dateForm" class="textbox" type="date" name="dateForm">
+    `;
+    },
+    destroy() {
+        document.querySelector('#candidateDetailsForm').innerHTML = '';
+    },
+};
+
+interviewApp.TechnicalLevelPickerHeader = function (option) {
+    const levelTitle = option.headers.map(element => `
```

Comment:
Bonus points if you can express this in a more succinct manner without `map` + `join`.